st2client/st2client/commands/pack.py

```diff
@@ -194,6 +195,10 @@ def __init__(self, resource, *args, **kwargs):
                                  action='store_true',
                                  default=False,
                                  help='Force pack installation.')
+        self.parser.add_argument('--no_deps',
```

Comment:
```suggestion
        self.parser.add_argument('--no-deps',
```

st2 CLI argument should be `--no-deps` to be consistent with existing naming where `-` is used as a separator instead of `_`.

```sh
$ st2 pack install --help
usage: st2 pack install [-h] [-t TOKEN] [--api-key API_KEY] [-j] [-y]
                        [-w WIDTH [WIDTH ...]] [--attr ATTR [ATTR ...] | -d]
                        [--python3] [--force]
                        pack [pack ...]

Install new packs.

positional arguments:
  pack                  Name of the packs in Exchange, or a git repo URL.

optional arguments:
  --api-key API_KEY     Api Key for user authentication. Get ST2_API_KEY from
```

It's however fine to use `_` in code/jinja/yaml and other places.